resources/analysisTools/rnaseqworkflow/featureCountsDexseq_2_FpkmTpm

```diff
@@ -79,9 +79,18 @@ if (length $gtf_file > 0) {
     chomp $line;
     my @line_split = split ("\t", $line);
     if ($line_split[2] eq "gene") {
-      my @comment = split ("\"",$line_split[8]);
-      $gtf{$comment[1]}="$line_split[0]\t$line_split[3]\t$line_split[4]\t$comment[1]\t.\t$line_split[6]\t$comment[9]";
-      $gtf_names{$comment[1]}=$comment[9];
+      my @comment = split (";",$line_split[8]);
```

Comment:
Unfortunately, I am not sure what kind of string is supposed to be parsed here. Can you add an example before the parser-code. That would help a lot.

Reply:
I have added an example in the new subroutine code annotation.